src/screens/OtherProfile/index.js

```diff
@@ -229,7 +214,7 @@ const OtherProfile = () => {
       user_id_follower: profile.myProfile.user_id,
       user_id_followed: other_id,
       username_follower: profile.myProfile.username,
-      username_followed: username,
+      username_followed: params?.data?.username,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The use of `params?.data?.username` instead of the previously set local state `username` is a good change for consistency.



<!-- This is an auto-generated comment by OSS CodeRabbit -->